### PR TITLE
feat(rgbpp-sdk/ckb): Update examples for SPV proof

### PR DESCRIPTION
**Pull request by duanyytop**
## Main Changes

- Update examples for SPV proof
- Rename the examples subdirectory from `src` to `local` and `queue` subdirectory will be added later

## Tips

Please note that some examples need to be tested in two steps. The first step is to send a BTC transaction, and the second step requires the confirmation of the BTC transaction. 

For details, please refer to the annotations of the example code.

## Reviewers
@Flouse @ahonn @ShookLyngs

### CODE REVIEW

File: examples/rgbpp/local/3-btc-transfer.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import { transactionToHex, sendRgbppUtxos, DataSource, ECPair, bitcoin, NetworkType } from '@rgbpp-sdk/btc';
+import { BtcAssetsApi } from '@rgbpp-sdk/service';
+
+// CKB SECP256K1 private key
+const CKB_TEST_PRIVATE_KEY = '0x0000000000000000000000000000000000000000000000000000000000000001';
+// BTC SECP256K1 private key
+const BTC_TEST_PRIVATE_KEY = '0x0000000000000000000000000000000000000000000000000000000000000001';
+// https://btc-assets-api.testnet.mibao.pro/docs/static/index.html
+const BTC_ASSETS_API_URL = 'https://btc-assets-api.testnet.mibao.pro/';
+const BTC_ASSETS_TOKEN = '';
```
Comment (Flouse) [inline examples/rgbpp/local/3-btc-transfer.ts:19]:
```suggestion
// API docs: https://btc-assets-api.testnet.mibao.pro/docs
const BTC_ASSETS_API_URL = 'https://btc-assets-api.testnet.mibao.pro';
```

File: examples/rgbpp/local/4-btc-jump-ckb.ts [hunk 2]
```diff
@@ -10,24 +9,21 @@ import {
...
 // CKB SECP256K1 private key
 const CKB_TEST_PRIVATE_KEY = '0x0000000000000000000000000000000000000000000000000000000000000001';
 // BTC SECP256K1 private key
 const BTC_TEST_PRIVATE_KEY = '0x0000000000000000000000000000000000000000000000000000000000000001';
-// https://btc-assets-api-develop.vercel.app/docs/static/index.html
-const BTC_ASSETS_API_URL = 'https://btc-assets-api-url';
-// https://btc-assets-api-develop.vercel.app/docs/static/index.html#/Token/post_token_generate
+// https://btc-assets-api.testnet.mibao.pro/docs/static/index.html
+const BTC_ASSETS_API_URL = 'https://btc-assets-api.testnet.mibao.pro/';
 const BTC_ASSETS_TOKEN = '';
```
Comment (Flouse) [inline examples/rgbpp/local/4-btc-jump-ckb.ts:26]:
```suggestion
// API docs: https://btc-assets-api.testnet.mibao.pro/docs  
const BTC_ASSETS_API_URL = 'https://btc-assets-api.testnet.mibao.pro';  
```

File: examples/rgbpp/local/5-spend-btc-time-cell.ts [hunk 3]
```diff
@@ -3,17 +3,18 @@ import {
...
+import { BtcAssetsApi } from '@rgbpp-sdk/service';
 
 // CKB SECP256K1 private key
-const CKB_TEST_PRIVATE_KEY = '0x0000000000000000000000000000000000000000000000000000000000000001';
-// See https://github.com/ckb-cell/ckb-bitcoin-spv-service#json-rpc-api-reference
-const SPV_SERVICE_URL = 'https://ckb-bitcoin-spv-service.testnet.mibao.pro';
+const CKB_TEST_PRIVATE_KEY = '';
+// https://btc-assets-api.testnet.mibao.pro/docs/static/index.html
+const BTC_ASSETS_API_URL = 'https://btc-assets-api.testnet.mibao.pro/';
+const BTC_ASSETS_TOKEN = '';
```
Comment (Flouse) [inline examples/rgbpp/local/5-spend-btc-time-cell.ts:17]:
```suggestion
// API docs: https://btc-assets-api.testnet.mibao.pro/docs  
const BTC_ASSETS_API_URL = 'https://btc-assets-api.testnet.mibao.pro';  
```

**Review by Flouse — APPROVED** [review]

**Comment by Dawn-githup** [comment]
- [ ] Reuse the Omiga protocol to quickly issue XUDT assets on CKB
- ckb address: ckt1qyqr9duhmee9095agxwqc5jz9xqqa4lucq0szkz35k
- Xudt has been minted with tx hash: 0x2e7a2db473b92c06eba9aaf2cc229029c54a366b68e46d3eef3ba4ac27feb430
- [ ] Jump XUDT from CKB to BTC
```
- BTC UTXO TxId: 
- e61fab10c391e331c37f31aae6591c0fca225795549cbefd43052132fcab06e1
```
- ckb address:  ckt1qyqr9duhmee9095agxwqc5jz9xqqa4lucq0szkz35k
- Rgbpp asset has been jumped from CKB to BTC and tx hash is::0x6fb6089ee94f795bee7eb1a323e9d77fa4f2039866b7b6d21451cb464267843e
- [ ] Transfer RGBPP asset on BTC
```
- BTC UTXO TxId: 
e61fab10c391e331c37f31aae6591c0fca225795549cbefd43052132fcab06e1
```
```
- BTC TxId:  b9c28f50c13833917406bbe7d330d0eee4afad7752c7495b322e87e54df1071b
- to btc address:  tb1qm06rvrq8jyyckzc5v709u7qpthel9j4d9f7nh3
```
- Rgbpp asset has been transferred on BTC and tx hash is 0xa3953c03744be80fed2f7af1b02a5daf7a54bda8bd5e9585642e1b7f2d4b2b94

- [ ] Jump RGBPP asset from BTC to CKB
```
- BTC TxId:  b9c28f50c13833917406bbe7d330d0eee4afad7752c7495b322e87e54df1071b
```
- ckb address:  ckt1qyqr9duhmee9095agxwqc5jz9xqqa4lucq0szkz35k
- btc address:  tb1qm06rvrq8jyyckzc5v709u7qpthel9j4d9f7nh3
- BTC TxId:  a463a5b9d28dc4935a217f2e1f60d727b4abcb0308e017ea07ed93e7e6e0b234
- gbpp asset has been jumped from BTC to CKB and tx hash is 0x68f652ad70393d983aa288eae80ad8b065653136c1f70d4f74dff3496e974c4c

**Pull request merged**